checkov/common/graph/checks_infra/base_check.py

```diff
@@ -23,7 +24,7 @@ def __init__(self) -> None:
         self.solver: Optional[BaseSolver] = None
         self.guideline: Optional[str] = None
         self.benchmarks: Dict[str: list] = {}
-        self.bc_severity: Optional[str] = None
+        self.bc_severity: Optional[Severity] = None
         self.pc_severity: Optional[str] = None
```

Comment:
I left this one because it is going to go away in favor of just one severity soon